x-pack/plugins/observability/public/pages/alerts/containers/alerts_page/alerts_page.tsx

```diff
@@ -109,12 +109,12 @@ function AlertsPage() {
         http,
       });
       // Note that the API uses the semantics of 'alerts' instead of 'rules'
```

Comment:
Would be great if we can remove this comment for us now that everything is referencing "rules"!